File: integration/example/3-nested-modules/repo/qux/src/Qux.scala
package qux
import scalatags.Text.all._
object Qux {
  val value = p("today")

  def main(args: Array[String]): Unit = {
    println("Foo.value: " + foo.Foo.value)
    println("Bar.value: " + bar.Bar.value)
    println("Qux.value: " + qux.Qux.value)
  }
}

File: example/kotlinlib/builds/4-nested-modules/foo/qux/src/qux/Qux.kt
package foo.qux

import com.github.ajalt.clikt.core.CliktCommand
import com.github.ajalt.clikt.parameters.options.option
import com.github.ajalt.clikt.parameters.options.required
import foo.bar.main as barMain
import kotlinx.html.p
import kotlinx.html.stream.createHTML

class Qux: CliktCommand() {
    val barText by option("--bar-text").required()
    val quxText by option("--qux-text").required()

    override fun run() {
        main(barText, quxText)
    }
}

fun main(barText: String, quxText: String) {
    barMain(barText)

    val value = createHTML(prettyPrint = false).p { text(quxText)  }.toString()
    println("Qux.value: $value")
}

fun main(args: Array<String>) = Qux().main(args)
